pkg/exporters/slack.go

```diff
@@ -171,6 +170,16 @@ func (s *Slack) parseSlackChannelConfig(channels string) error {
 // Match namespaces from service IDs to Slack channels.
 func (s *Slack) determineChannels(message msg.Message) []string {
 	var channels []string
+	// When flux delete objects there are no ServiceIDs
```

Comment:
I guess this function could be refactored?? IIUC we could have a single for loop to determine the slack channels?